index.js

```diff
@@ -626,14 +626,21 @@ class BitbucketScm extends Scm {
     * @return {Promise}
     */
     _getCheckoutCommand(config) {
-        const checkoutUrl = `https://${config.host}/${config.org}/${config.repo}`;
+        const checkoutUrl = `${config.host}/${config.org}/${config.repo}`;
+        const sshCheckoutUrl = `git@${config.host}:${config.org}/${config.repo}`;
         const checkoutRef = config.prRef ? config.branch : config.sha;
         const command = [];
 
         // Git clone
         command.push(`echo Cloning ${checkoutUrl}, on branch ${config.branch}`);
+        command.push('if [ ! -z $SCM_CLONE_TYPE] && [ $SCM_CLONE_TYPE = ssh ]; ' +
```

Comment:
missing a space after `! -z $SCM_CLONE_TYPE`
Also, could `! -z` be replaced with `-n`?